### ISSUE TITLE
Unable to start the debugger in "Run Extension" mode

### ISSUE THREAD

**Issue by timvancleef** [nicoespeon/abracadabra#69]
First of all: Thanks for this great extension 👍

I'm experiencing an issue when trying to start the debugger in "Run Extension" mode and i'm not sure if this is a bug or just my somewhat misconfigured VSCode 😅

<img width="565" alt="Bildschirmfoto 2020-01-16 um 12 08 44" src="https://user-images.githubusercontent.com/7040078/72520476-d67d4480-3859-11ea-8b86-941d2cbc8a46.png">

VSCode shows me an error telling me that the `npm: build` task does not exist:

<img width="1150" alt="Bildschirmfoto 2020-01-16 um 12 08 57" src="https://user-images.githubusercontent.com/7040078/72520485-daa96200-3859-11ea-8681-e11f6cab39be.png">

The [`.vscode/tasks.json`](https://github.com/nicoespeon/abracadabra/blob/master/.vscode/tasks.json) indeed only contains an `npm: watch` task and i'm not sure if the build task is actually missing or if VSCode "somehow" should figure it out.

Extension version: latest master commit (a72c6f9)
VSCode version: Version: 1.41.1

**Comment by nicoespeon** [comment]
Hmmmm interesting, let's try to solve this.

So first thing: I got rid of the watch tasks. They don't work well yet, so let's drop it until we make it work correctly. We should still be able to build the project with the "Run Extension" configuration.

Now for this issue, I was relying on the [Task Autodetection](https://code.visualstudio.com/docs/editor/tasks#_task-autodetection) feature of VS Code. It would make `npm: build` run the `npm run build` command, defined in the package.json. 

I can see that could be turned off, which is probably your case.

![image](https://user-images.githubusercontent.com/1094774/72635056-e8c6b380-3929-11ea-8d58-858c9e67735e.png)

In that scenario, I got the same error:

![image](https://user-images.githubusercontent.com/1094774/72634997-c9c82180-3929-11ea-8bfc-1f645e93da04.png)

Thus, I just added a task for `npm: build`. So it should work now, even with autodetect turned off 🎉 

Let me know if you still have an issue. Thanks for reporting 😉
